apps/webview-ui/src/app/globals.css

```
@@ -186,3 +186,9 @@
     cursor: pointer;
   }
 }
+
+@custom-variant can-hover {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@custom-variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)